src/main/java/seedu/sellsavvy/logic/parser/DeleteOrderCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package seedu.sellsavvy.logic.parser;
+
+import static seedu.sellsavvy.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.logic.commands.ordercommands.DeleteOrderCommand;
+import seedu.sellsavvy.logic.commands.personcommands.DeletePersonCommand;
+import seedu.sellsavvy.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteOrderCommand object
+ */
+public class DeleteOrderCommandParser implements Parser<DeleteOrderCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteOrderCommand
+     * and returns a DeleteOrderCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public DeleteOrderCommand parse(String args) throws ParseException {
+        try {
+            Index index = ParserUtil.parseIndex(args);
+            return new DeleteOrderCommand(index);
+        } catch (ParseException pe) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeletePersonCommand.MESSAGE_USAGE), pe);
```

Comment:
why is it DeletePersonCommand.Message?

Reply:
Agreed, please verify your imports as well per this change.